modules/caddyhttp/serverless/testdata/pyechoserver/app.py

```
@@ -0,0 +1,73 @@
+        logger.info(f"[{datetime.now().isoformat()}] Preparing response data")
+        response_data = {
+            "headers": headers,
+            "body": body.decode('utf-8') # Assuming UTF-8, adjust if other encodings are expected
+        }
+
+        logger.info(f"[{datetime.now().isoformat()}] Setting request_handled flag to True")
+        request_handled = True
+
+        # Schedule shutdown after response is sent
+        logger.info(f"[{datetime.now().isoformat()}] Scheduling server shutdown")
+        import asyncio
```

Pylint (3.3.7)

[convention] Import outside toplevel (asyncio)

(C0415)